### ISSUE TITLE
dotnet restore fails if CLI is installed in a path with spaces

### ISSUE THREAD

**Issue by Mpdreamz** [dotnet/cli#911]
Getting the following when calling `dotnet restore` on https://github.com/elastic/elasticsearch-net/tree/vnext/dnx-support/src/Elasticsearch.Net

> Mpdreamz@DESKTOP C:\Users\Mpdreamz\dev\elastic\coreclr\src\Elasticsearch.Net
> $ dotnet restore
> 
> Unhandled Exception: System.IO.FileNotFoundException: Could not load file or assembly 'Program' or > one of its dependencies. The system cannot find the file specified.

**Comment by davidfowl** [comment]
Run `dotnet -v restore`

**Comment by Mpdreamz** [comment]
> Mpdreamz@DESKTOP C:\Users\Mpdreamz\dev\elastic\coreclr\src\Elasticsearch.Net
> $ dotnet -v restore
> Running C:\Program Files\dotnet\bin\corerun.exe C:\Program 
> Files\dotnet\bin\NuGet.CommandLine.XPlat.dll restore --runtime win7-x86 --runtime win7-x64
> Process ID: 10680
> 
> Unhandled Exception: System.IO.FileNotFoundException: Could not load file or assembly 'Program' or > one of its dependencies. The system cannot find the file specified.

**Comment by Ikarus76** [comment]
Hi, I had the same issue. It seems to be an issue with the "program files" path. After installing the dotnet binaries to 'c:\dotnet' the problem is solved.

**Comment by davidfowl** [comment]
ah, spaces in the path bug it looks like.

**Comment by blackdwarf** [comment]
@brthor is this the same root cause as #525?

**Comment by brthor** [comment]
@blackdwarf @Ikarus76 @Mpdreamz This looks like the same issue as #525, I am working on the fix

**Comment by blackdwarf** [comment]
OK, I'm renaming this to make it easier to find.

**Comment by GuardRex** [comment]
The command of interest (thanks @alexkoeplinger) is ...
```
msiexec /i "msi path" TARGETDIR="C:\myfolder" /qb
```

**Comment by wjk** [comment]
:+1: This happens to me too. Not only that, I installed the .NET CLI on other computers in the past (using the same link as above), and it worked before, so this is in addition a regression. @brthor I noticed that you placed the RC2-Feb milestone on this issue. Does this mean that a fix won't be forthcoming until that date? This is a really inconvenient bug for me to have to deal with. Is there any chance it could be fixed sooner? Thanks!
